Jenkinsfile.promote

```diff
@@ -147,7 +147,7 @@ pipeline {
         //             String nextVersion = getNextVersion()
```

Comment:
why is this section commented?

Reply:
`That part is commented as it messes with PRs on operatorhub which may require some changes`